market_analysis/conftest.py

```diff
@@ -0,0 +1,166 @@
+import os
+import pytest
+from pyramid import testing
+from .models import (
+    get_engine,
+    get_session_factory,
+    get_tm_session,
+)
+from .models.meta import Base
+import transaction
+from market_analysis.scripts.test_db import STOCKS_100
+from .models.mymodel import Users, Association, Stocks
+from passlib.apps import custom_app_context as pwd_context
+
+OS_USER = os.environ.get('USER')
+DB_SETTINGS = {
+                #'sqlalchemy.url': 'postgres://{}:@localhost:5432/testing'
+                'sqlalchemy.url': 'postgres:///testing'
+                .format(OS_USER)
+                }
+
+USER_CREDENTIALS = {'username': 'fake', 'password': 'fake'}
+USER_PASS_HASH = pwd_context.encrypt(USER_CREDENTIALS['password'])
+ADMIN_CREDENTIALS = {'username': 'admin', 'password': 'admin'}
+ADMIN_PASS_HASH = pwd_context.encrypt(ADMIN_CREDENTIALS['password'])
```

Comment:
good work building global constants you'll use throughout your test fixtures.  That's a great idea.